### DISCUSSION TITLE
Website layout is messy

### DISCUSSION THREAD

**Discussion by UE2lab** [community/community#69770] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi everyone! 

i am here to find answers related to my messy website layout. I don't know why suddenly my web layout became messy. Here i provided the capture of my before page and after it became messy layout. Now, i am still finding the way to solve it but i havent got any. So, I hope i can get the answers from you guys! Thank you !!

![after](https://github.com/community/community/assets/124585923/5bae99f7-2958-4929-beed-6350f5287d38)
![before](https://github.com/community/community/assets/124585923/eaf399be-46b1-46ef-98ca-658d13d82bf0)

**Comment by ebubekirgungor** [comment]
Hello, your css links in your html file have github url like: "https://ue2lab.github.io/UE2lab/css/main.css"
This github url redirects to your website but without https. So when your index.html loaded, it loads your css files under http. 
To fix this, you should change all "https://ue2lab.github.io/UE2lab/" texts to "https://ue2lab.korea.ac.kr/" in your index.html file.
Or you can try to change your github.io redirecting to https://ue2lab.korea.ac.kr/

**Reply by UE2lab** [reply]
Hello! Thank you for assisting me~
However, I found a problem in searching for the index.html file. It seems I don't have any file named "index.html" Could you guide me on how to find it?
Thank you.

**Reply by ebubekirgungor** [reply, marked as answer]
Hello, I  thought that you are using index.html file. I found the problem is in the config file. In https://github.com/UE2lab/UE2lab/blob/gh-pages/_config.yml you should change baseurl to "" and url to https://ue2lab.korea.ac.kr/

**Reply by UE2lab** [reply]
Fantastic! Thank you so much, I fixed it by following your guidance. I hope you have a great day!

**Reply by ebubekirgungor** [reply]
You are welcome :) Can you select my answer as answered?
